package.json

```diff
@@ -165,6 +165,11 @@
           "default": "ghcr.io/ledgerhq/ledger-app-builder/ledger-app-dev-tools:latest",
           "description": "Ledger developer tools Docker image."
         },
+        "ledgerDevTools.dockerRunArgs": {
+          "type": "string",
+          "default": "",
+          "description": "Additional arguments to pass to the 'docker run' command for the Ledger developer tools Docker image."
```

Comment:
You should explicitly define the scope of this setting : `"scope": "resource",` so that it does go in the `.vscode/settings.json` of the app that's being worked on, otherwise it will apply (I guess) to all folders of the workspace.

Reply:
Done in b47a1a23540891afee4c91ba90cab3b2dd2823ae